lib/fake_stripe/bootable.rb

```diff
@@ -0,0 +1,18 @@
+require "capybara"
+require "capybara/server"
+require "fake_stripe/utils"
+
+module Bootable
+  def boot(port = FakeStripe::Utils.find_available_port)
+    instance = new
+    Capybara::Server.new(instance, port: port).tap(&:boot)
+  end
+
+  def boot_once
+    @boot_once ||= boot(server_port)
```

Comment:
So `#boot` takes an optional `port` which defaults to `FakeStripe::Utils.find_available_port`, but in `#boot_once` we explicitly pass `#server_port`, which returns `FakeStripe::Utils.find_available_port`. How about

```suggestion
    @boot_once ||= boot
```

instead? Do we need `#server_port`?

Reply:
I don't think we can do that, since the server port in

```rb
STRIPE_CONNECT_HOST = "http://localhost:#{FakeStripe::StubStripeConnect.server_port}"
```
needs to match the server port we boot with in `boot_once`.

I thought about changing boot to `def boot(port = server_port)`, but that probably doesn't make sense either since then boot would break if we called it twice without an argument.

We may not actually need a public boot method. I brought it over because it existed on the original class, but I realize now it is not tested, documented, or used in any obvious way in the code base.